### PR TITLE
Add video related UDTs

### PR DESCRIPTION
**Pull request by changhiskhan**

### CODE REVIEW

File: python/rikai/vision.py [hunk 1]
```diff
@@ -167,3 +171,49 @@ def to_numpy(self) -> np.ndarray:
             "bbox": self.bbox.to_numpy(),
             "score": self.score,
         }
+
+
+class VideoStream:
```
Comment (eddyxu) [inline python/rikai/vision.py:176]: Move these two classes to "rikai.types.vision" or `rikai.types.video`?

File: python/rikai/vision.py [hunk 2]
```diff
@@ -167,3 +171,49 @@ def to_numpy(self) -> np.ndarray:
...
+    def __repr__(self) -> str:
+        return f"VideoStream(uri={self.uri})"
+
+    def _repr_html_(self):
+        """TODO: codec"""
+        from IPython.display import Video
+
+        if pathlib.Path(self.uri).exists():
+            path = pathlib.Path(self.uri).relative_to(os.getcwd())
+            return Video(path, width=480, height=320)._repr_html_()
```
Comment (eddyxu) [inline python/rikai/vision.py:193]: will the `width` and `height` be configurable ?

**Review by eddyxu — APPROVED** [review]
LGTM

File: python/rikai/vision.py (hunk 2 above)
Comment (changhiskhan) [reply python/rikai/vision.py:193]: we can add a rikai.options.display module that allows the user to configure this in the notebook. I was planning to do it in a separate PR

File: python/rikai/vision.py (hunk 1 above)
Comment (changhiskhan) [reply python/rikai/vision.py:176]: ok i can rebase on your changes

File: python/rikai/vision.py [hunk 3]
```diff
@@ -115,3 +120,49 @@ def to_numpy(self) -> np.ndarray:
...
+
+    def __repr__(self) -> str:
+        return f"VideoStream(uri={self.uri})"
+
+    def _repr_html_(self):
+        """TODO: codec"""
+        from IPython.display import Video
+
+        if pathlib.Path(self.uri).exists():
+            path = pathlib.Path(self.uri).relative_to(os.getcwd())
```
Comment (eddyxu) [inline python/rikai/vision.py:141]: it misses a few imports, i.e., `os`, `pathlib`

**Pull request merged**